domains/ftolinkbyharry.localplayer.dev.json

```diff
@@ -0,0 +1,14 @@
+{
+    "domain": "localplayer.dev",
+    "subdomain": "ftolinkbyharry",
+
+    "owner": {
+        "email": "hariprasad15032003@gmail.com"
+    },
+
+    "record": {
+        "NS": ["ns1.hostry.com"]
```

Comment:
We do not support new NS records as per our readme.md guidelines.

Reply:
Your files have not been changed, records are still invalid. This convo should not have been resolved.